routers/web/web.go

```diff
@@ -232,10 +232,17 @@ func RegisterRoutes(m *web.Route) {
 	// Routers.
 	// for health check
 	m.Get("/", Home)
-	m.Get("/.well-known/openid-configuration", user.OIDCWellKnown)
-	if setting.Federation.Enabled {
-		m.Get("/.well-known/nodeinfo", NodeInfoLinks)
-	}
+	m.Group("/.well-known", func() {
+
```

Comment:
I think this empty new line can be removed.

Reply:
indeed